session/bootstrap.go

```diff
@@ -1330,6 +1334,36 @@ func IsUpgradingClusterState(s Session) (bool, error) {
 	return stateInfo.State == syncer.StateUpgrading, nil
 }
 
+func checkOrSyncUpgrade(s Session, ver int64) {
+	if ver < SupportUpgradeHTTPOpVer {
+		terror.MustNil(SyncUpgradeState(s))
+		return
```

Comment:
No need to set `SyncNormalRunning`?

Reply:
Notify the cluster that the upgrade is complete by occurring `upgrade/finish`, and this request will send to the new TiDB(>= v7.4)